### DISCUSSION TITLE
Removal of Fire Immunity from MC and BWL bosses.

### DISCUSSION THREAD

**Discussion by plasma-93** [chromiecraft/chromiecraft#3574] [Suggestions]
Fire Immunity isn't fun, it's cool for like your first MC for the theme, and at every point after that it's just lame. 

**The change to allow MC and BWL mobs to be Bled and Poisoned should have also included a removal of their Fire Immunity, and instead have high fire resistance.**

"Blizzlike isn't always the best design" is obviously something the devs agree with me on, given the change for Bleed Immunity and Poison Immunity, and I don't think it's going too far to have Rag be High Fire Res instead of Fire Immune. 

It would increase build diversity, allow people to play the specs they actually want to play, and fit better with the servers casual vibe imo. Wrath boss design included high resistance, but never immunity to elements, this was on purpose, and in early Cata, they removed spell school immuties due to the lack of interesting design 

I don't think this change has any negative impacts, it would just be more fun.

**Comment by RestoCC** [comment]
Yes.

**Comment by Annamaria-CC** [comment]
Bleed immunity was only changed because that was a tbc/wotlk change.

**Reply by plasma-93** [reply]
And the reason it was changed, is because it was lame

**Comment by KaneoHunter** [comment]
With high fire resist ur still gonna play frost bro

**Comment by Annamaria-CC** [comment]
Thanks for your suggestion.

Previous changes to mechanics were are related if it was ever a thing in vanilla/tbc/wotlk. Sometimes we chose one mechanic over the other if it fits progression and/or QoL improvement. Bleed immunity is one of those things that was removed from them in TBC. This suggestion does not fall under this behaviour and we do not see a reason to change that.

Locking this suggestion.
